meerk40t/gui/laserpanel.py

```diff
@@ -225,6 +225,23 @@ def spooler_lookup(self, *args):
             self.combo_devices.Append(spools[i])
         self.combo_devices.SetSelection(index)
         self.button_save_file.Enable(hasattr(self.context.device, "extension"))
+        self.set_pause_color()
+
+    def set_pause_color(self):
+        new_color = None
+        new_caption = _("Pause")
+        try:
+            if self.context.device.driver.paused:
```

Comment:
This isn't consistent for all drivers. Though in an attribute test so it won't matter much.

Reply:
It is for most now, ruida the most notable exception